daemon-scheduler/Makefile

```diff
@@ -34,6 +34,11 @@ $(LOCAL_BINARY): $(SOURCES)
 	./scripts/build_binary.sh ./out
 	@echo "Built Blox Daemon Scheduler"
 
+$(SWAGGER_GEN): $(SWAGGER_SPEC)
+	./scripts/generate_swagger_artifacts.sh
+	PATH="$(ROOT)/scripts:${PATH}" go generate ./copyright_gen/...
```

Comment:
Is this line even needed? This seems to be a duplicate of the line in the `generate` target, and is only called by the `generate` target.